meterfs/meterfs.c

```diff
@@ -599,9 +688,15 @@ int meterfs_readFile(id_t id, off_t off, char *buff, size_t bufflen, meterfs_ctx
 	unsigned int idx;
 	size_t chunk, i = 0, pos = off;
 
+	if (bufflen <= 0)
```

Comment:
```suggestion
	if (bufflen == 0)
```
`bufflen` is type `size_t` and so `>= 0`.